cmd/debug.go

```
@@ -66,8 +66,8 @@ var debugGenRpzCmd = &cobra.Command{
 
 		fmt.Printf("Received %d bytes of data\n", len(resp.Msg))
 
-		fmt.Printf("black count=%d: %v\n", resp.BlacklistedNames)
-		fmt.Printf("grey count=%d: %v\n", resp.GreylistedNames)
+		fmt.Printf("deny count=%d: %v\n", resp.DenylistedNames)
```

golangci-lint (1.62.2)

printf: fmt.Printf format %d has arg resp.DenylistedNames of wrong type map[string]bool

(govet)